src/applications/vaos/services/mocks/index.js

```diff
@@ -32,6 +32,8 @@ const schedulingConfigurations = require('./v2/scheduling_configurations.json');
 const clinicsV2 = require('./v2/clinics.json');
 
 varSlots.data[0].attributes.appointmentTimeSlot = generateMockSlots();
+const mockAppts = [];
```

Comment:
I made a quick and dirty way to store submitted appointments in the mock server. They'll disappear whenever you change the mocks, but it at least means the transition from submit to detail page is less confusing.

Reply:
Nice, I think this is great.